compiler/natives/src/crypto/sha256/sha256.go

```diff
@@ -26,9 +26,25 @@ func webCryptoSum256(data []byte) (res [Size]byte, err error) {
 }
 
 func Sum256(data []byte) (res [Size]byte) {
-	if len(data) >= 2048 {
+	if len(data) >= 2600 {
 		// On big enough buffers, try to call the webcrypto implementation, it's much faster
-		// than the go implementation (typically 25x for a 10 MB buffer)
+		// than the go implementation. For small buffers though, webcrypto is much slower under Chrome,
+		// so we stay with the go implementation:
+		// +--------+-----------------------------------------------+
+		// |        |                Execution time                 |
+		// | Buffer +-----------------------+-----------------------+
+		// |        |       Chrome 55       |       Firefox 50      |
+		// | Length |-----------+-----------+-----------+-----------+
+		// |        |    Go     | webcrypto |    Go     | webcrypto |
+		// +--------+-----------+-----------+-----------+-----------+
+		// | 1B     | 83ns*     | 2.004ms   | 116ns*    | 149ns     |
+		// | 1kB    | 818ns*    | 2.003ms   | 334ns     | 152ns*    |
+		// | 2.5kB  | 2.072ms*  | 2.011ms   | 672ns     | 159ns*    |
+		// | 2.6kB  | 2.175ms   | 2.020ms*  | 689ns     | 159ns*    |
+		// | 2.7kB  | 2.400ms   | 2.022ms*  | 710ns     | 160ns*    |
+		// | 100kB  | 774ms     | 24ms*     | 215ms     | 16ms*     |
+		// | 10MB   | 1.632s    | 41ms*     | 1.76s     | 74ms*     |
+		// +--------+-----------+-----------+-----------+-----------+
```

Comment:
@r-l-x Please debug where exactly the latency happens. How long does it take for the promise to return vs. the Go code to continue running? I think this may be related to the blocking and channel operations.